plugins/amoeba/platforms/reference/tests/TestReferenceAmoebaVdwForce.cpp

```diff
@@ -436,12 +466,8 @@ void testVdwAlchemical(int power, double alpha, double lambda, AmoebaVdwForce::A
     for (int i=0; i<8; i++) {
         expectedForces[i] *= scale;
     }
-
-    double tolerance          = 1.0e-04;
-    compareForcesEnergy(testName, expectedEnergy, energy, expectedForces, forces, tolerance);
```

Comment:
I don't think you meant to delete this code, did you?  You just removed the part of this test case that did the actual testing.